packages/drivers/odsp-driver/src/getFileLink.ts

```diff
@@ -167,10 +174,18 @@ async function getFileItemLite(
             const fileItem = await getWithRetryForTokenRefresh(async (options) => {
                 attempts++;
                 const {siteUrl, driveId, itemId} = odspUrlParts;
-                const token = await getToken({ ...options, siteUrl, driveId, itemId});
+                const storageTokenFetcher = toInstrumentedOdspTokenFetcher(
+                    logger,
+                    odspUrlParts,
+                    getToken,
+                    true /* throwOnNullToken */,
+                );
+                const storageToken = await storageTokenFetcher(options,"GetSharingLinkTokenLite")
```

Comment:
This will have `_GetToken` appended at the end so let's call it to match the function name
```suggestion
                const storageToken = await storageTokenFetcher(options,"getFileItemLite");
```

Reply:
Also I think you should pass `true` for `alwaysRecordTokenFetchTelemetry` to maintain existing logging, here and above.